src/com/aca/items/PenAudioRecorder.java

```diff
@@ -0,0 +1,17 @@
+package com.aca.items;
+
+import com.aca.types.AudioRecorder;
+import com.aca.types.MemoryGadget;
+
+public class PenAudioRecorder extends MemoryGadget implements AudioRecorder {
+
+    public String color;
```

Comment:
restrict color value, don't allow to put here green or some other value, use here enum or add validation for its value